src/protocol/ErrorResponse.spec.ts

```diff
@@ -36,12 +36,16 @@ describe('Exception', () => {
     [ErrorCode.NoSuchNode]: undefined,
     [ErrorCode.NoSuchRequest]: undefined,
     [ErrorCode.NoSuchScript]: undefined,
+    [ErrorCode.NoSuchStoragePartition]: undefined,
```

Comment:
I don't see the new Errors added to `ErrorResponse.ts`

Reply:
Fixed